samples/wifi/radio_test/sample_description.rst

```diff
@@ -558,7 +559,53 @@ Testing
               [00:24:25.202,606] <inf> otp_prog: Written REGION_DEFAULTS (0x154) : 0xfffffffb
               [00:24:25.203,002] <inf> otp_prog: Finished Writing OTP params
 
-         See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands.
+
+         * To set a regulatory domain with the following configuration:
+
+           * Regulatory domain: US
+
+           Execute the following command:
+
+           .. code-block:: console
+
+              wifi_radio_test reg_domain US
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+				wifi_radio_test show_config
+
+				reg_domain = US
+
+         .. note::
+
+            * The default regulatory domain is 00 (World regulatory).
+
+		 * To bypass regulatory domain, set ``bypass_reg_domain`` to ``1`` using the following command:
+
+			* bypass_reg_domain: 1
+
+           Execute the following sequence of command:
+
```

Comment:
```suggestion
```

Reply:
This is probably needed.  (In line 590 a blank line is needed)